### DISCUSSION TITLE
FEAT: 'quick week' 7 days

### DISCUSSION THREAD

**Discussion by Symbiot78** [mealie-recipes/mealie#1202] [Feature Request]
**Before submitting this feature request I have**
_COMPLETE AND FILL THESE OUT_
- [x ] Reviewed the documentation and verified that my feature request isn't already a feature.
- [x ] Reviewed the existing feature requests and my feature has not been requested.
- [x ] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**

I'd like the quick week to have the option to cover a 7 day week and not just a 5 day week.

what parts of the problem are considered to be in-scope and out-of-scope._

**(Optional): Suggest A Solution**

Add config option to change the quick week to 7

* Details of the technical implementation
* Tradeoffs made in design decisions
* Caveats and considerations for the future

If there are multiple solutions, please present each one separately. Save comparisons for the very end.)

**Additional Information**
_FILL THESE OUT_
I don't have the skills to create the code myself.

**Comment by michael-genson** [comment]
Mealie has changed drastically since this discussion was opened, so I'm closing it since it's no longer relevant.
